modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.tsx

```diff
@@ -64,7 +64,11 @@ export default function LeftSidebar({setShowModal}: LeftSidebarProps) {
 			(key) => key === selectedObjectFolder.name
 		) as string;
 
-		setExpandedKeys(new Set([selectedObjectFolderKey]));
+		const filteredFolders = leftSidebarItems
+			.filter((item) => item.leftSidebarObjectDefinitionItems?.length)
+			.map((filteredNames) => filteredNames.name);
+
+		setExpandedKeys(new Set([selectedObjectFolderKey, ...filteredFolders]));
```

Comment:
This solution is making everything expanded at first render. The correct behavior is to have the selected folder expanded and rest collapsed, at first render. The right behavior is to have the selected folder expanded and the rest collapsed at first render. Expansion should initially controlled by the search matches when there is a search key entered.

Also, there is something wrong with `selectedObjectFolderKey`. If uncategorized is not selected, `selectedObjectFolderKey` is `undefined`, which adds `undefined` as the first item of `expandedKeys` array. This does  not look right and may be related to our problem.